test/unit/specs/components/Identity/getIdentity/createIdentityPayload.spec.js

```diff
@@ -24,12 +25,17 @@ describe("createIdentityPayload", () => {
     expect(payload.getUseIdThirdPartyDomain()).toBeTrue();
   });
 
+  it("returns false from getDocumentMayUnload", () => {
+    const payload = createDataCollectionRequestPayload();
+    expect(payload.getDocumentMayUnload()).toBeFalse();
+  });
+
   it("serializes properly", () => {
     const payload = createIdentityPayload(["ECID"]);
     payload.addIdentity("IDNS", {
       id: "ABC123"
     });
-    expect(payload.toJSON()).toEqual({
```

Comment:
This technically wasn't really serializing. `JSON.stringify` recursively calls `toJSON` on the content passed to it, which is a better representation of what we're trying to test.